### DISCUSSION TITLE
Make the Title sortable via RefinableString is not working / No values

### DISCUSSION THREAD

**Discussion by McMauser** [microsoft-search/pnp-modern-search#1892] [Q&A] [answered]
Hello together, I build a pretty cool solution with the Modern Search Web Part and I really appreciate all your hard work to realize such a cool custom web part!

Unfortunately, there is one small thing I can't fix myself and I don't know why this is not working.

I know that the standard Title mapped Property is not sortable. So I took my free RefinableString18 and mapped this to several crawled properties (OWS_TITLE, TITLE, TERMTITLE) but none of them worked. The RefinableString18 just gives me no value back - it is always empty.

My goal is to make the Search Result sorted by the Title by default. Therefore I need the sortable RefinableString18 with the values of the titles. 

Has someone also experienced this issue or has a tip how I can realize my plan?
How is it possible to map the Title to a sortable RefinableString?

I'm thankful for any kind of advise.

Best regards
Marek

**Comment by wobba** [comment, marked as answer]
Have you re-index your content or added a new item which has Title set in the library and checked after you did the mapping?

**Reply by McMauser** [reply]
Hi wobba,

I never tried the re-index in the site settings. That's a very good tip.
Let me check this and I will come back to you.

Thank you very much!

**Reply by wobba** [reply]
You always have to re-index if you changed a mapping affecting existing content.

**Reply by McMauser** [reply]
Thank you wobba. Re-index was actually what I'm looking for.
